mautic/mautic
# Pipedrive integration : add deal from mautic

Hi,

I've worked for **Kombiz** on extending the current Pipedrive Integration plugin with deal handling.

#### Steps to test this PR:
1. pull the code
2. update schema (apply doctrine migration)
3. as admin in plugins/pipedrive : check deal feature (if not done yet, you also need a working pipedrive integration setup) 
4. run the mautic:integration:pipedrive:fetch command to fetch informations about pipelines, stages products and deals
 5. now you can create a form or campaign "push to integration action", and by selecting Pipedrive, you can tell mautic to create a deal from that action (and thus select pipeline/stage, product, etc).

Sidenote : In Pipedrive, products can have several prices. We hope to be able to improve the form as shown below in a future evolution, in order to help the user see those prices after selecting a product. As of now, user can put an arbitrary price in an regular input field.

![form](https://user-images.githubusercontent.com/10561580/28784153-389adf5a-761b-11e7-8ef1-3a8d21bb56ee.png)
